osc-tools/src/main/resources/quality/OSC_checks.xml

```diff
@@ -77,9 +77,6 @@
     <module name="ModifiedControlVariable"/>
     <module name="GenericWhitespace"/>
   </module>
-  <module name="FileLength">
```

Comment:
Removing file length check. Default file length violation is 2000 lines. However, ReleaseUpgradeMgr.java is already causing checkstyle violation and continues to grow.

Reply:
http://checkstyle.sourceforge.net/config_filters.html#SuppressionCommentFilter#SuppressionCommentFilter please look at using that or using existing supression mechanism

https://github.com/opensecuritycontroller/osc-core/blob/master/osc-tools/src/main/resources/quality/checkstyle-suppressions.xml